src/sentry/options/manager.py

```diff
@@ -270,6 +270,12 @@ def isset(self, key: str) -> bool:
 
         return self.is_set_on_disk(key)
 
+    def set_on_db(self, key: str) -> bool:
+        """
+        Check if a key has been set on the database.
```

Comment:
```suggestion
    def set_in_db(self, key: str) -> bool:
        """
        Check if a key has been set in the database.
```

Values get set *in* databases, and *on* disk.